engines/director/detection_tables.h

```diff
@@ -4644,6 +4645,10 @@ static const DirectorGameDescription gameDescriptions[] = {
 
 	MACGAME1("mathtest", "", "mathtest (mac)", "cdb27c916044ae4dceb4b7326063fa03", 301925, 400),
 
+	// Demo version of "Maths Workshop", published by Broderbund (1994-1995)
+	// The full game is not made with Director, but with MOHAWK (eriktorbjorn)
+	WINDEMO1t("mathsworkshop", "Preview", "_MATHDAT.EXE", "5306745f0734b6cbd94e4567a0b5805a", 696931, 400),
```

Comment:
That is the old macro, for new entries we generally use this style
```suggestion
	WINDEMO1("mathsworkshop", "Preview", "_MATHDAT.EXE", "t:5306745f0734b6cbd94e4567a0b5805a", 696931, 400),
```